ripley/api/canvas_site_controller.py

```diff
@@ -86,7 +86,7 @@ def edit_sections(canvas_site_id):
     return tolerant_jsonify(
         {
             'jobId': job.id,
-            'jobStatus': job.get_status(),
+            # 'jobStatus': job.get_status(),
```

Comment:
@lyttam Do we need to screen out this field for some reason?

Reply:
@pauline2k I removed the commented-out line for now. The job status has gotten complicated since there are two jobs now - the queued job that performs the sis import, and the `BcoursesProvisionSiteJob` job that is kicked off by the first job. I want to figure out how to monitor the status of both.

Reply:
@lyttam Makes sense, thanks!